app/validation/validator.py

```diff
@@ -193,6 +195,29 @@ def validate_duplicates(self, json_to_validate, ignored_keys, special_key):
 
         return duplicate_errors
 
+    def validate_duplicate_options(self, json_to_validate):
+        errors = []
+
+        for block in self._get_blocks(json_to_validate):
+            answers_for_block = self._get_answers_for_block(block)
+
+            for answers in answers_for_block:
+                labels = set()
```

Comment:
any reason this cant just be `labels = []`

Reply:
@jonnyshaw89 Since I am adding after each option, no need to add the duplicates. Would work either way but any reason not to have it as `set()` as `sets` are faster for searching I believe?